packages/gestalt/src/Contents.js

```diff
@@ -207,10 +208,17 @@ class Contents extends Component<Props, State> {
       return { top: null, height: null };
     }
 
-    const { scrollBoundaryContainerRef, positionRelativeToAnchor } = this.props;
+    const {
+      __dangerouslyIgnoreScrollBoundaryContainerSize,
+      scrollBoundaryContainerRef,
+      positionRelativeToAnchor,
+    } = this.props;
 
     // Define the height based the reference to render: ScrollBoundaryContainer or screen viewport
-    const height = scrollBoundaryContainerRef?.offsetHeight ?? window.innerHeight ?? 0;
+    let height = window.innerHeight ?? 0;
+    if (!__dangerouslyIgnoreScrollBoundaryContainerSize) {
+      height = scrollBoundaryContainerRef?.offsetHeight ?? window.innerHeight ?? 0;
```

Comment:
I'm curious, why do we use `scrollBoundaryContainerRef?.offsetHeight` instead of  `window.innerHeight`, can't we just use `window.innerHeight`?

Reply:
```
    let height = window.innerHeight ?? 0;
    if (!__dangerouslyIgnoreScrollBoundaryContainerSize) {
      height = scrollBoundaryContainerRef?.offsetHeight ?? height; >> redundant
```

Reply:
maybe 

```
    let height = window.innerHeight ?? 0;
    if (!__dangerouslyIgnoreScrollBoundaryContainerSize && scrollBoundaryContainerRef?.offsetHeight ) {
      height = scrollBoundaryContainerRef?.offsetHeight
```

Reply:
Either of these works. I have a slight preference for the first one but not that strong.

a nit on the second logic is that optional chaining wouldn't be necessary after verifying the existence of scrollBoundaryContainerRef in the conditional

Reply:
@AlbertCarreras 
We handled this prop based on `scrollBoundaryContainer` height because in some cases/screens on Pinboard when we define the size based on screen, the Popover renders under the TopBar. And we not have a case(tracked) where the scrollBoundaryContainer is under the TopBar; 

PS:. The zIndex is not fixing the Popover visibility to these cases. e.g. `/pin-builder`